.pipelines/templates/.builder-release-template.yaml

```diff
@@ -164,6 +164,21 @@ steps:
     condition: always()
     displayName: Check Build Performance
 
+  - task: PublishPipelineArtifact@0
```

Comment:
aint those two tasks inversed ? don' ty ou need to copy and publish after ? Maybe I' dont have the full logic :)

Reply:
They are not dependent on one another. The copy task puts it in the staging directory to be published with the bulk artifacts folder at the end of the pipeline, so its convenient to find most of the time. 

But that bulk artifacts folder is only published if the build succeeds, so the publish task before it is necessary in order to make sure its published even if the build fails, since thats when we're most interested in it. @djsly

Reply:
this is an artifact of that fact that we actually publish every artifact twice, after we abandoned a previous change since it required a larger refactor, we can clean this up